src/layman/layer/geoserver/tasks.py

```diff
@@ -42,15 +44,16 @@ def refresh_wms(
     if self.is_aborted():
         raise AbortedException
 
+    coverage_store_name = wms.get_geotiff_store_name(layername)
     if file_type == settings.FILE_TYPE_VECTOR:
+        gs_util.delete_coverage_store(geoserver_workspace, settings.LAYMAN_GS_AUTH, coverage_store_name)
```

Comment:
I believe this line is extra and should be removed, because if we do PATCH with `file`, coverage store and coverage are deleted synchronously.